From 90acde398acf8a691cdcb9cffcadd0affccb7b59 Mon Sep 17 00:00:00 2001
From: Stelios Tsampas <loathingkernel@gmail.com>
Date: Thu, 23 Apr 2026 00:55:09 +0300
Subject: [PATCH 13/14] Revert "HACK: steam: wine.inf: Associate the steam
 protocol with steam.exe."

This reverts commit eeb10d71bea438089b62906f0aaa40be1a05dc97.
---
 loader/wine.inf.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index 4e39507f9d6..f010afe2c91 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -313,7 +313,6 @@ HKCR,ftp\shell\open\command,,2,"""%11%\winebrowser.exe"" ""%1"""
 HKCR,http\shell\open\command,,2,"""%11%\winebrowser.exe"" ""%1"""
 HKCR,https\shell\open\command,,2,"""%11%\winebrowser.exe"" ""%1"""
 HKCR,mailto\shell\open\command,,2,"""%11%\winebrowser.exe"" ""%1"""
-HKCR,steam\shell\open\command,,,"""%16426%\Steam\Steam.exe"" -- ""%1"""
 
 [ContentIndex]
 HKLM,System\CurrentControlSet\Control\ContentIndex\Language\Neutral,"WBreakerClass",,"{369647e0-17b0-11ce-9950-00aa004bbb1f}"
-- 
2.54.0

